app/Annotations/OpenApi/controllersAnnotations/studentLanguagesDetailAnnotations/AnnotationsStudentLanguagesDetail.php

```diff
@@ -11,8 +13,8 @@ class AnnotationsStudentLanguagesDetail
      *      path="/students/{uuid}/languages",
      *      operationId="getStudentLanguages",
      *      tags={"Languages"},
-     *      summary="Get a detailed list of languages and level for a student",
-     *      description="Returns a list of Languages and level for a specific student. (Note: This endpoint returns hardcoded project details)",
+     *      summary="Get a detailed list of languages and their respective level from a specific student's resume",
```

Comment:
It's seems a little bit repetitive. Don't you think the "summary" could be shorter and then go longer on the description?

Reply:
You're right. They are too similar and don't serve their purpose. I've modified the summary to be more concise, and the description to be more descriptive.